turbopack/crates/turbopack-tests/tests/execution/turbopack/tree-shaking/basic/issues/unexpected export __star__-b847a0.txt

```diff
@@ -0,0 +1,4 @@
+warning - [analysis] [project]/turbopack/crates/turbopack-tests/tests/execution/turbopack/side-effects-optimization/basic/input/node_modules/package-star/not-executed.js  unexpected export *
+  export * used with module [project]/turbopack/crates/turbopack-tests/tests/execution/turbopack/side-effects-optimization/basic/input/node_modules/package-star/not-executed.js [test] (ecmascript) which has no exports
```

Comment:
Should I fix it?
`export * from './not-executed'` exists, and we load it with `ModulePart::proxied_export("a")` and `ModulePart::proxied_export("b")`, so I think this warning is correct.

Reply:
`turbopack/crates/turbopack-tests/tests/execution/turbopack/side-effects-optimization/basic` reports no issues, so I think it should be able to handle that without issues too.